bin/get-hetzner-cloud-token

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Scripts adapted from ones mentioned in https://github.com/ansible/molecule/pull/1811#issuecomment-487449174.

Reply:
@webknjaz, I am sure you will have something to say regarding these scripts, so I ping you now.

Reply:
I'm normally putting a space for readability after `#!` and use another invocation style. But no strong reason to changing this.
```suggestion
#! /usr/bin/env bash
```